Documentation/upgrade.md

```diff
@@ -4,47 +4,68 @@ weight: 60
 ---
 
 # Upgrades
-This guide will walk you through the manual steps to upgrade the software in a Rook cluster from one version to the next.
-Rook is a distributed software system and therefore there are multiple components to individually upgrade in the sequence defined in this guide.
-After each component is upgraded, it is important to verify that the cluster returns to a healthy and fully functional state.
+This guide will walk you through the manual steps to upgrade the software in a Rook cluster from one
+version to the next. Rook is a distributed software system and therefore there are multiple
+components to individually upgrade in the sequence defined in this guide. After each component is
+upgraded, it is important to verify that the cluster returns to a healthy and fully functional
+state.
 
-This guide is just the beginning of upgrade support in Rook.
-The goal is to provide prescriptive guidance and knowledge on how to upgrade a live Rook cluster and we hope to get valuable feedback from the community that will be incorporated into an automated upgrade solution by the Rook operator.
+With the release of Rook 0.9, significant progress has been made toward the goal of incorporating
+an automated upgrade solution into the Rook operator. While significant, this has merely lain the
+foundation for automated upgrade support, and we will continue to be open to community feedback for
+further improving and refining upgrade automation.
 
 We welcome feedback and opening issues!
 
 ## Supported Versions
-The supported version for this upgrade guide is **from a 0.7 release to a 0.8 release**.
+The supported version for this upgrade guide is **from a 0.8 release to a 0.9 release**.
 Build-to-build upgrades are not guaranteed to work.
 This guide is to test upgrades only between the official releases.
 
-For a guide to upgrade previous versions of Rook, please refer to the version of documentation for those releases.
+For a guide to upgrade previous versions of Rook, please refer to the version of documentation for
+those releases.
+- [Upgrade 0.7 to 0.8](https://rook.io/docs/rook/v0.8/upgrade.html)
 - [Upgrade 0.6 to 0.7](https://rook.io/docs/rook/v0.7/upgrade.html)
 - [Upgrade 0.5 to 0.6](https://rook.io/docs/rook/v0.6/upgrade.html)
 
 ### Patch Release Upgrades
-The changes in patch releases are scoped to the minimal changes necessary and are expected to be straight forward to upgrade.
-For upgrades of 0.8 patch releases such as `0.8.0` to `0.8.1`, see the [patch release upgrade guide](upgrade-patch.md).
+One of the goals of the 0.9 release is that patch releases are able to be automated completely by
+the Rook operator. It is intended that upgrades from one patch release to another are as simple as
+updating the image of the Rook operator. For example:
+```
+kubectl -n rook-ceph-system set image deploy/rook-ceph-operator rook-ceph-operator=rook/ceph:v0.9.x
```

Comment:
might be nice to use the specific tag for an example, such as `v0.9.1`

Reply:
I suppose that's probably okay. I just checked to see what would happen if a user inputs something here that doesn't have an image, and the old operator isn't terminated until the new one is starting. When I do `kubectl -n rook-ceph-system set image deploy/rook-ceph-operator rook-ceph-operator=rook/ceph:v0.9.x`, the 'new' operator is showing `ErrImagePull`, so we might as well make this something that we don't have to change in the future.